test/unit/classification_test.rb

```diff
@@ -314,8 +302,9 @@ def setup
                           :value => "345",
                           :use_puppet_default => true
     end
-    enc = classification.enc
-    refute enc['base'].has_key?(key.key)
+
+    enc = @host.puppetclass_parameters
+    refute enc['base'].key?(key.key)
```

Comment:
We have it only in tests, otherwise Rubocop would complain about it in the build process. Since I touched this file, I have changed it here too.